crates/subspace-wasm-tools/src/lib.rs

```diff
@@ -19,18 +42,24 @@ pub fn create_runtime_bundle_inclusion_file(
 
     // Create a file that will include execution runtime into consensus runtime
     let profile = env::var("WBUILD_PROFILE").expect("Set above; qed");
-    let execution_wasm_bundle_path = format!(
-        "{target_dir}/{profile}/wbuild/{runtime_crate_name}/{}.compact.wasm",
-        runtime_crate_name.replace('-', "_")
-    );
+    let execution_wasm_bundle_path = target_dir
+        .join(profile)
+        .join("wbuild")
+        .join(runtime_crate_name)
+        .join(format!(
+            "{}.compact.wasm",
+            runtime_crate_name.replace('-', "_")
+        ));
```

Comment:
This just does more allocations and requires you to use `to_string_lossy` in the end

Reply:
Yeah right, but I think it is better to abstract directory delimiter. I guess performance is not a priority here, right?

Reply:
`/` works just fine on Windows in 99% of cases, so I usually don't bother

Reply:
Having more types seems right, instead of using format strings for paths :(
If you insist I can remove it.